test/unit/test-custom-element.js

```diff
@@ -1156,7 +1156,9 @@ describes.realWin('CustomElement', {amp: true}, (env) => {
           expect(element2.sizerElement.style.paddingTop).to.equal('1%');
         });
 
-        it('should rediscover sizer to apply heights in SSR', () => {
+        // TODO BEFORE MERGE: is this test still valid? Why should this ever be "rediscovered".
```

Comment:
@dvoytenko: do you know in which cases we are meant to rediscover a sizer / if this is still a valid need? This test fails with the new InOb path